pnpm-lock.yaml

```diff
@@ -72,8 +72,8 @@ importers:
         specifier: ^0.4.2
         version: 0.4.4
       sbc-common-components:
-        specifier: 3.0.8
-        version: 3.0.8
+        specifier: 3.0.12
+        version: 3.0.12
```

Comment:
I'm not sure why the old lock file specified v3.0.8, because the previous package file specified v3.0.12. Anyways, looks correct now.
🤷‍♂️